coral/src/domain/user/user-api.ts

```diff
@@ -30,10 +31,23 @@ async function getUserList(
     );
 }
 
+async function getUser(): Promise<User> {
```

Comment:
We mix that a bit 🤔  It's our `user` domain, so I felt `getUser` is more fitting than `getMyProfileInfo`. "getMyProfileInfo" is information about our backend structure \ api (that was build very close to Angular frontend) that does not really need to bleed into frontend. 

Maybe `getUserProfile` would be a good compromise? It relates to our UI but keeping the domain.

Reply:
I think it's either no change, or any name we think is more meaningful ^^ But I think it doesn't really matter if backend information bleeds into the front end in this way. It would matter if we mapped UI exactly to the data structure of the backend, but I don't think we do this. Changing the naming for handlers which care only wrapping a calll to the API client only introduces ambiguity, for me.

I think the valid usecase for having an API handler name that is different is when we process the data in some way, or if we are doing several calls in one handler.

But I don't feel very strongly about it ^^

Reply:
Then maybe we leave it as it is for now? I think we're going to touch those names anyway as soon as we start to align APIs to aiven :D and then we can still see how to establish a more consistent pattern.

Reply:
Sure ^^

Reply:
Ok! I update the branch & pipeline is green, so you can merge if you want to